www.foia.gov/_sass/modules/_foia_request_form.scss

```diff
@@ -1,7 +1,16 @@
-// TODO Temporary foia-legacy overrides
+$hide-tabs-width: 768px;
```

Comment:
We should move this variable up to `base` or something to avoid having the same variable in two different files.